### DISCUSSION TITLE
Add setting to enable auto focus when something is logged

### DISCUSSION THREAD

**Discussion by BertvanHoekelen** [spatie/ray#155] [Ideas]
The first version of ray automatically focused when something was logged. This was removed in version [1.0.3](https://spatie.be/products/ray/release-notes#1.0.3).

I really liked this function because it saves me a keystroke. It would be great if this could be a setting so that it can be turned on and off. 

<br /><br />

Thanks for the awesome app guys! Really enjoying it.

**Comment by freekmurze** [comment]
We'll add this behind a preference setting soon!
